docs/src/main/sphinx/connector/sql-update-limitation.fragment

```diff
@@ -0,0 +1,32 @@
+### UPDATE
+
+If a ``WHERE`` clause is specified, the ``UPDATE`` operation only works if the
+predicate in the clause can be fully pushed down to the data source.
+
+Only ``UPDATE`` statements with constant assignments and predicates are
+supported. For example, the following statement is supported:
+
+```sql
+UPDATE table SET col1 = 1 WHERE col3 = 1
+```
+
+Arithmetic expressions can not be used with the ``SET`` command. The following
```

Comment:
```suggestion
Arithmetic expressions cannot be used with the ``SET`` command. The following
```

Reply:
Also .. turn sentence around

The following statement is not supported, since arithmetic ..